mmv1/products/compute/ResourcePolicy.yaml

```diff
@@ -336,7 +333,7 @@ properties:
         conflicts:
           - group_placement_policy.0.max_distance
         immutable: true
-        min_version: 'beta'
+        ignore_read: true
```

Comment:
Can you confirm that `gpuTopology` is not returned from v1 API, but is returned from beta API?

Reply:
I checked that manually and API doesn't return the value. That's probably issue on the API side.